### DISCUSSION TITLE
Eviction policy for memcached mode

### DISCUSSION THREAD

**Discussion by sbkoto** [dragonflydb/dragonfly#3133] [General]
What is the eviction policy for memcached mode?  in the metrics I see only expires. 
Is it possible to tune it and set it like a redis? LRU?

**Comment by romange** [comment]
tt's not possible, Dragonfly has a single policy, which is the same for Redis and Memcache mode.

**Reply by sbkoto** [reply]
okay, how set single policy?

```
set maxmemory-policy allkeys-lru
-ERR Out of memory
```

**Reply by romange** [reply]
Please refer to [these docs](https://www.dragonflydb.io/docs/managing-dragonfly/flags#--cache_mode)
